src/Core/Checkout/Cart/Order/Api/OrderRecalculationController.php

```diff
@@ -20,6 +22,9 @@
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Attribute\Route;
 
+/**
+ * @codeCoverageIgnore
+ */
```

Comment:
why? there is quite some logic in this controller.

Reply:
Because there's currently no tests for it. And I changed one exception being thrown and now am punished to write tests for this file. If you insist, I'll do it, but only if you insist.

Reply:
As discussed privately, the coverage job is not required. Yes, I would always appreciate a test, but it's not required and @codeCoverageIgnore is definitely not the way to go. Only for things like Structs, DomainException classes and things, that do not need testing.